aws/data_source_aws_rds_orderable_db_instance.go

```diff
@@ -195,6 +203,24 @@ func dataSourceAwsRdsOrderableDbInstanceRead(d *schema.ResourceData, meta interf
 		input.EngineVersion = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("default_only"); ok && v.(bool) {
+		input := &rds.DescribeDBEngineVersionsInput{
+			DefaultOnly: aws.Bool(true),
+			Engine:      input.Engine,
+		}
+
+		result, err := conn.DescribeDBEngineVersions(input)
+		if err != nil {
+			return fmt.Errorf("error reading RDS default engine version: %w", err)
+		}
+
+		if len(result.DBEngineVersions) < 1 {
+			return fmt.Errorf("no RDS default engine version found for engine: %v", aws.StringValue(input.Engine))
+		}
+
+		input.EngineVersion = result.DBEngineVersions[0].EngineVersion
```

Comment:
Rather than introducing a second API call, lets allow the existing API call and downstream logic tease out any issues:

```suggestion
		input.DefaultOnly = aws.Bool(true)
```

Reply:
This would be excellent except `DescribeOrderableDBInstanceOptionsInput` does not have a `DefaultOnly` field. 😞